src/routes/_components/status/StatusMediaAttachments.html

```diff
@@ -43,8 +45,18 @@
     background: none;
   }
 
+  .inner-div {
```

Comment:
I disable Svelte's CSS encapsulation (for perf, to reduce the size of the CSS selectors), so in the future we probably want to avoid too-generic class names